.github/workflows/docs.yml

```diff
@@ -2,7 +2,7 @@ name: Docs
 
 on:
   push:
-    branches: [main, dev]
+    branches: [main, dev, docs-describe-zarp-outputs]
```

Comment:
@fgypas I need to remove this before merging. I used it to deploy the documentation in the website.